tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -57,7 +57,6 @@ def perform_broker_upgrade(self, to_version):
             node.version = KafkaVersion(to_version)
             self.kafka.start_node(node)
 
-    @ignore
```

Comment:
This test is still using `sleep` and I would assume it might be unstable -- what should have beed the reason why we disabled the test -- I think, we should rather rewrite it that just enable it.

Reply:
As I said in #5605 maybe we can merge as is and re-write in a follow-on PR.